api/helpers/add-game-to-match.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
   fn: async ({ game }, exits) => {
     try {
       if (!game.isRanked) {
-        return exits.success();
+        const match = sails.helpers.mockCasualMatch(game);
```

Comment:
Ultimately I'd like to rework the db so that casual games have a real match as well. But the idea here is to keep score during rematch games even for casual games like this
<img width="635" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/18bc6ad3-f3f7-46c3-82e6-970dce9dbe15">

Which for consistency's sake we use the same data structure that the ranked matches do: an object w/ a list of the games along w/ some metadata about who's playing etc

Reply:
Yeah exactly. Upon further thought and based on your suggestion, I think we can just pretend it's a real match and use the name `getCasualMatch()`